api/src/pcapi/core/external/attributes/api.py

```diff
@@ -69,10 +69,68 @@ def update_external_pro(email: str | None) -> None:
             )
 
 
+def get_anonymized_attributes(user: users_models.User) -> models.UserAttributes | models.ProAttributes:
+    if user.has_pro_role:
+        attributes = get_pro_attributes(user.email)
+        attributes.is_pro = True
+        attributes.is_user_email = False
+        attributes.user_id = user.id
```

Comment:
incohérent avec la ligne au-dessus

Reply:
Pourquoi as-tu marqué comme résolu sans changer ni commenter ?
A priori si `is_user_email` est `False` alors il ne peut pas y avoir de `user_id`, qui d'ailleurs n'aura plus cette adresse email. Je suggère donc : 
```suggestion
        attributes.user_id = ""
```

Reply:
Je me suis pris les pieds dans le tapis dans le tapis avec git... je corrige